src/sections/requirements.tex

```diff
@@ -1,99 +1,233 @@
+        U06         & Als Praxismitarbeiter möchte ich eine Rückmeldung erhalten, wenn eine Benachrichtigung nicht versendet werden kann, damit Benachrichtigungen nicht verloren gehen. & F03 \\
         \hline
-        U03         & ALS Benutzer WILL will nur Notifikationen sehen die für mich relevant sind, DAMIT nicht unnötig belästigt werde. & F0x \\
+        U07         & Als Praxismitarbeiter möchte ich auswählen können an welchem Gerät ich das Praxisrufsystem verwende und die dafür erstellte Konfiguration erhalten, damit das Praxisrufsystem optimal verwendet werden kann. & F05 \\
         \hline
-        U04         & ALS Benutzer WILL ich über eingehende Notifikationen aufmerksam gemacht werden, DAMIT ich Notifikationen nicht übersehe. & F0x \\
+        U8          & Als Praxismitarbeiter möchte ich einen physischen Knopf am Behandlungsstuhl haben damit ich notifikationen darüber versenden kann. & F07 \\
         \hline
-        U05         & ALS Benutzer WILL ich sehen welche Notifikationen ich verpasst habe, DAMIT ich vergessenes nachholen kann. & F0x \\
+        U9          & Als Praxismitarbeiter möchte ich, dass mir Benachrichtigungen vorgelesen werden, damit ich informiert werde, ohne meine Arbeit unterbrechen zu müssen. & F08 \\
+        \hline
+        U10          & Als Praxismitarbeiter möchte ich einen anderen Client anrufen können damit Fragen direkt geklärt werden können. & F09 \\
+        \hline
+        U11         & Als Praxismitarbeiter möchte ich Unterhaltungen mit mehreren anderen Clients gleichzeitig führen können damit komplexe Fragen direkt geklärt werden können. & F10 \\
         \hline
     \end{tabular}\label{tab:userstories1}
 \end{table}
 
-\subsubsection*{Admin}
+\clearpage
+
+\subsubsection*{Praxisverantwortlicher}
 
 \begin{table}[h]
     \centering
     \begin{tabular}{|l|p{13cm}|c|}
         \hline
-        \textbf{Id} & \textbf{Anforderung}                                                                                                                                 & \textbf{Features} \\
+        \textbf{Id} & \textbf{Anforderung}                                                                                                                                                     & \textbf{Features} \\
         \hline
-        U10         & ALS Administrator WILL mehrere Geräte verwalten kann DAMIT jedes Gerät für Verwendungsort optimiert werden kann. & F0x \\
+        U12         & Als Praxisverantwortlicher möchte ich mehrere Geräte verwalten können, damit jedes Gerät für Verwendungsort optimiert ist. & F0x \\
         \hline
-        U11         & ALS Administrator WILL ich definieren können welches Gerät, welche Anfragen versenden kann DAMIT jedes Gerät für Ort und Zweck optimiert werden kann & F0x \\
+        U13         & Als Praxisverantwortlicher möchte ich definieren können welches Gerät, welche Anfragen versenden kann, damit jedes Gerät Verwendungsort optimiert ist. & F0x \\
```

Comment:
Allenfalls U12 und U13 zusammefassen zu: "Als Praxisverantwortlicher möchte ich für jedes Gerät eine eigene Konfiguration erstellen, damit jedes Gerät die Bedürfnisse des Praxismitarbeiters erfüllt" ?

Reply:
Ich verstehe was du meinst. Das war die Absicht von U13. Ich würde gerne separat aufführen, dass der Kunde das Bedürfnis hat mehrere Clients haben zu können.

U12 -> Es gibt mehrere Clients
U13 -> Die können Unterschiedlich konfiguriert sein. 

So können wir sagen, das wir im POC nur U12 umsetzen. Da sind mehrere Clients unterstützt, aber sie verhalten sich alle gleich. 

Ich passe das "DAMIT" des U12 an, auf "damit das Praxisrufsystem in mehreren Zimmern gleichzeitig genutzt werden kann." Ist fast trivial, aber man soll bei Requirements ja keine annahmen treffen ^^ 

Passt das so für dich?